spec/models/category_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe Category do
+  it "saves itself" do
+    first_category = Category.create name: "The category of love"
```

Comment:
Not a big deal but you can use parens after `create`; imo it makes your code more readable. `first_category = Category.create(name: "The category of love"`)